style.css

```diff
@@ -0,0 +1,35 @@
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+}
+
+.hamburger-menu {
+  border-color: transparent;
+}
+
+.hamburger-menu:hover {
+  cursor: pointer;
+}
+
+.hamburger-menu:active {
+  cursor: context-menu;
+}
+
+.headline {
+  background-image: url('./img/header-background-mobile.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  font-style: normal;
+  font-size: 28px;
+}
```

Comment:
- [ ] Please make sure that this content is aligned with flexbox. :pray: 
  * This gives us a lot of options to align all our elements, plus, it's responsive.